app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -116,7 +116,8 @@ class DefaultToolbarMenu(
                 highlight = if (hasAccountProblem) {
                     BrowserMenuHighlightableItem.Highlight(
                         imageResource = R.drawable.ic_alert,
-                        backgroundResource = R.color.sync_error_background_color
+                        backgroundResource = R.color.sync_error_background_color,
+                        colorResource = R.color.sync_error_background_color
```

Comment:
Shouldn't this be `sync_error_text_color`? Otherwise the image would be the same colour as the background right?

Reply:
@jonalmeida the `ic_alert` drawable has the tint `sync_error_text_color`. The `colorResource` is simply used for the "dot" that appears on the menu icon when the menu is collapsed to notify the user there is a warning inside.

Reply:
Updated 😄